Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -254,6 +254,14 @@ public static void Show(MixedRealityControllerMapping controllerMapping, Seriali
             defaultFieldWidth = EditorGUIUtility.fieldWidth;
         }
 
+        public static void RepaintWindow()
```

Comment:
nit: missing summary (though, it looks like this entire class is...)

Reply:
I'll do some maintenance :)